### DISCUSSION TITLE
Turn-on / turn-off inverter using WEB API

### DISCUSSION THREAD

**Discussion by Robyxxx** [tbnobody/OpenDTU#730] [General]
I'm new with this fantastic tool. I have a TSUN M350, perfectly working. I'm interesting in turn-on / turn-off the inverter using WEB API. It is possible?

**Comment by Engehausen** [comment]
Try something like:
```
r = requests.post(url = f'http://{dtu_ip}/api/power/config', data = f'data={{"serial":"{serial0}", "power": {0}}}', auth = HTTPBasicAuth(dtu_nutzer, dtu_passwort), headers = {'Content-Type': 'application/x-www-form-urlencoded'})
print(f'Power off ({r.json()["type"]})')
```

This is Python code. I hope this helps.

**Comment by fellpower** [comment]
How does a get / post should look like?

im not using python, im using iobroker to switch off the inverter ^^
